Unable to compile for Windows

There is reference to a `sql_backend` variable that I think existed in a previous version but was refactored away.

Accompanying pull request to fix this coming momentarily.

```rs
error[E0425]: cannot find value `sql_backend` in this scope
   --> C:\Users\Jacob\scoop\persist\rustup\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\tabiew-0.9.0\src\main.rs:121:66
    |
121 | ...                   match execute(action, &mut app, &mut sql_backend) {
    |                                                            ^^^^^^^^^^^ not found in this scope

error[E0061]: this function takes 2 arguments but 3 arguments were supplied
   --> C:\Users\Jacob\scoop\persist\rustup\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\tabiew-0.9.0\src\main.rs:121:35
    |
121 | ...                   match execute(action, &mut app, &mut sql_backend) {
    |                             ^^^^^^^                   ---------------- unexpected argument #3
    |
note: function defined here
   --> C:\Users\Jacob\scoop\persist\rustup\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\tabiew-0.9.0\src\handler\action.rs:135:8
    |
135 | pub fn execute(action: AppAction, app: &mut App) -> AppResult<Option<AppAction>> {
    |        ^^^^^^^
help: remove the extra argument
    |
121 -                             match execute(action, &mut app, &mut sql_backend) {
121 +                             match execute(action, &mut app) {
    |

Some errors have detailed explanations: E0061, E0425.
For more information about an error, try `rustc --explain E0061`.
error: could not compile `tabiew` (bin "tw") due to 2 previous errors
error: failed to compile `tabiew v0.9.0`, intermediate artifacts can be found at `C:\Users\Jacob\AppData\Local\Temp\cargo-installizHuQn`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
```

--- a/src/main.rs
+++ b/src/main.rs
@@ -118,7 +118,7 @@ fn start_tui(tabs: Vec<(DataFrame, String)>, script: String, history: History) -
                     if matches!(key_event.kind, KeyEventKind::Press | KeyEventKind::Repeat) {
                         let mut action = keybind.action(app.context(), key_event);
                         loop {
-                            match execute(action, &mut app, &mut sql_backend) {
+                            match execute(action, &mut app) {
                                 Ok(Some(next_action)) => action = next_action,
                                 Ok(_) => break,
                                 Err(err) => {

